challenges/deploy-WebV/README.md

```diff
@@ -0,0 +1,78 @@
+# Deploy Web Validate
+
+## Background
+
+Web Validate (WebV) is an end-to-end test tool that we will use to generate web requests against our application. These web requests can be validated on a simple returned status code or can be configured to perform a deep validation. Crucially, the logs and metrics of the testing can be stored in Graphana or Azure Logs to ensure application functionality and performance through integration and load testing. Although not part of this challenge, dashboards and monitors are created from these logs and metrics to gain a real time view of the application along with its historical baseline.
```

Comment:
you can't store logs in Graphana - it's just a visualization tool

Reply:
done